### PR TITLE
webserver: fix live testing wallet issues

### LINKED ISSUE #1949: ui: markets view not working for unsupported asset
Use **client/webserver/live_test.go**, `go test -tags lgpl,live -timeout 60m`, and MONA-KMD market for testing.

image

image

### PR DESCRIPTION
**Pull request by ukane-philemon**
- This fixes an error on the markets page for an unsupported wallet.
- Also fixes panic when trying to create specific test wallets(`mona`,...).
- Minor refactoring of the live tests. 
Closes #1949

### CODE REVIEW

**Comment by chappjc** [comment]
> * Also fixes panic when trying to create specific wallets.

Will you expand on this?  Where's the panic?  In Go?

File: client/webserver/live_test.go [hunk 1]
```diff
@@ -366,6 +354,16 @@ var tExchanges = map[string]*core.Exchange{
...
+			"mona": {
+				ID:    22,
+				Confs: 5,
+				Amt:   1e10,
+			},
+			"vtc": {
+				ID:    28,
+				Confs: 5,
+				Amt:   1e11,
+			},
```
Comment (ukane-philemon) [inline client/webserver/live_test.go:366]: @chappjc, sorry I was not too clear with my comments. Creating a `mona` test wallet will panic here when trying to pay reg fees cause it was not previously available. https://github.com/decred/dcrdex/blob/178daeb258c377a4c9318a41e7be7ae7e6f7fbba/client/webserver/live_test.go#L1408

**Review by buck54321 — APPROVED** [review]
Looks great!

**Comment by chappjc** [comment]
> Looks great!

Thanks for reviewing that, @buck54321

**Pull request merged**